docs/troubleshooting/harvester.md

```diff
@@ -4,7 +4,37 @@ sidebar_label: Harvester
 title: ""
 ---
 
-## Generate a support bundle
+## Fail to Deploy Multi-node Cluster due to Incorrect HTTP Proxy Setting
+
+### Configure HTTP Proxy During Harvester Installation
+
+In some environments, you configure [HTTP Proxy](../airgap/#configure-an-http-proxy-during-installation) during Harvester installation.
```

Comment:
This setting will only take effect during the ISO installation and will not apply to the underneath RKE2 services directly. So I was wondering if it is because users have added the system `http-proxy` setting that causes the problem.

Reply:
updated. The user configured `http-proxy` in Harvester GUI, then the issue came.

Seems we did not highlight/automatically add the NODE IP/CIDR into `noProxy`.